app/Http/Controllers/API/Profile/UploadAvatarController.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Brewmap\Http\Controllers\API\Profile;
+
+use Brewmap\Http\Controllers\Controller;
+use Brewmap\Http\Requests\Profile\UpdateProfileAvatarRequest;
+use Brewmap\Services\StoreFileService;
+use Illuminate\Http\JsonResponse;
+
+class UploadAvatarController extends Controller
+{
+    private $pathToAvatar;
```

Comment:
I think it might be a local variable even, because StoreFileService, which is currently the only provider of avatar path value, is only injected in the 'store' method scope.

Also, as a future bonus (inb4 yagni), exposing an interface instead of concrete StoreFileService sounds like a cool thing to do here, because current implementation is bound to underlying local storage. It could make integration testing easier too.

Reply:
@kamilpiech97, why didn't you address @HerringTheCoder comments?